public/javascript/pump/view.js

```diff
@@ -630,6 +631,11 @@
                 streams.notifications = view.minorStreamView.model;
             }
             return streams;
+        },
+        closeDropdown: function(e) {
+            // Close by trigger because call function 'dropdown'
+            // in element parent or child, block navigation
+            $("#fat-menu.dropdown.open").trigger("click");
```

Comment:
Ohh, yep. Thanks. Been a while since I used jQuery.

I still feel like there's a better way to do this but offhand I don't know what it is so can you add a line that says `// TODO find a way to do this directly instead of synthesizing an event`?